src/lizaalert/courses/models.py

```diff
@@ -441,14 +441,14 @@ class Subscription(TimeStampedModel):
     enabled - признак активности записи на курс.
     """
 
-    class Flag(models.TextChoices):
-        ACTIVE = 1, "Запись активна"
-        INACTIVE = 0, "Запись не активна"
+    class Status(models.TextChoices):
+        ENROLLED = "Enrolled", "Запись активна"
+        NOTENROLLED = "NotEnrolled", "Запись не активна"
```

Comment:
```suggestion
        NOTENROLLED = "not_enrolled", "Запись не активна"
```

Reply:
названия статусов или с большой буквы, или разделить _

```suggestion
        NOT_ENROLLED = "not_enrolled", "Запись не активна"
```